aspnet-core/Core-Concepts.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Core Concepts | ASP.NET Core | Syncfusion
-description: Core Concepts
+title: Core Concepts in ASP.NET Core Common | Syncfusion
```

Comment:
This is not a common control, change in all places.

Reply:
Done